test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -2344,6 +2348,9 @@ TEST_F(ClusterManagerImplTest,
 }
 
 TEST_F(ClusterManagerImplTest, DynamicAddedAndKeepWarmingWithoutTlsCertificateEntity) {
+  TestScopedRuntime scoped_runtime;
+  Runtime::LoaderSingleton::getExisting()->mergeValues(
```

Comment:
Do we have tests for the old behavior when this is disabled?

Reply:
added.